demos/dotnet/dotnet/project.json

```diff
@@ -0,0 +1,12 @@
+﻿{
+  "dependencies": {
```

Comment:
Could you then add a readme note to the folder explaining what all these projects are for? 
Otherwise looks good.